README.md

```diff
@@ -1,70 +1,42 @@
-# Getting Started with Create React App
+# Project Name
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+>mathmagician .
```

Comment:
- [ ] Please add a proper description for your project here to make your readme file more professional.